### ISSUE TITLE
[Bug] Mana bar and mana number 

### ISSUE THREAD

**Issue by Petar25** [Fix3dll/SkyblockAddons#60]
### Bug Description

Mana bar and mana number sometimes lag behind the actual mana.

### SBA Version

1.9.0-b.16unofficial

### Other Installed Mods

```md
# System Stats
[OS][Windows 10]
[CPU][12x 12th Gen Intel(R) Core(TM) i5-12400]
[Display][1920x1080 (NVIDIA Corporation)]
[GPU][NVIDIA GeForce RTX 3060/PCIe/SSE2]
[GPU Driver][4.6.0 NVIDIA 560.94]
[Maximum Memory][16125MB]
[Shaders][TRUE]
# Java Stats
[Java][1.8.0_51 64bit]
[Memory][ 42% 1749/4096MB]
[Memory Allocated][ 58% 2400MB]
# Game Stats
[FPS][75]
[Loaded Mods][42]
[Forge][11.15.1.2318]
[Optifine][TRUE]
# Neu Settings
[On SkyBlock][TRUE]
[Mod Version][2.3.3+b23d0a27]
[Version Id][20303]
[SB Profile][Grapes]
# Repo Stats
[Last Commit][30aba0173401b9a3b441869cfa4958952a13511f]
[Repo Location][https://github.com/NotEnoughUpdates/NotEnoughUpdates-REPO/tree/master]
[Using Backup][false]
[Loaded Items][6619]
[Lowest Bin API Location][moulberry.codes]
# Repo Files
[.gitattributes]
[.github][1]
[.gitignore]
[.vscode][2]
[constants][33]
[enforced_values][2]
[items][6619]
[LICENSE]
[mobs][220]
[neu-items-master.zip]
[README.md]
[update.json]
[update.json.asc]
# Mods Loaded
[Minecraft Coder Pack][minecraft.jar (9.19)]
[Forge Mod Loader][forge-1.8.9-11.15.1.2318-1.8.9.jar (8.0.99.99)]
[Minecraft Forge][forge-1.8.9-11.15.1.2318-1.8.9.jar (11.15.1.2318)]
[Dungeons Guide][minecraft.jar (unknown)]
[PolyPatcher][minecraft.jar (1.10.1)]
[Essential][Essential (forge_1.8.9).processed.jar (1.3.4.1)]
[OneConfig][OneConfig (1.8.9-forge).jar (0.2.2-alpha213)]
[Hypixel Mod API][HypixelModAPI-1.0.1.1.jar (1.0.1.1)]
[SkyblockPersonalized][1.8.9.SkyblockPersonalized.1.7.0.0.jar (1.7.0.0)]
[Autotip][Autotip-3.1.1 (1.8.9_forge).jar (3.1.1)]
[Chatting][Chatting-1.8.9-forge-2.0.6.jar (2.0.6)]
[Craftify][Craftify-1.8.9-1.15.0-forge.jar (1.15.0)]
[CrashPatch][CrashPatch-1.8.9-forge-2.0.1.jar (2.0.1)]
[ChatTriggers][ctjs-2.2.1-1.8.9.jar (2.2.1-1.8.9)]
[Dungeons Guide Loader][dungeonsguide-4.0.0-beta11.0-standalone.jar (4.0.0)]
[Fullscreen Windowed][FullscreenWindowed-1.10.2-1.5.1.jar (1.5.1)]
[Hypixel Autocomplete Mod][Hypixel Autocomplete-1.1 (1.8.9).jar (LOCAL)]
[Hytils Reborn][Hytils Reborn-1.8.9-forge-1.7.3.jar (1.7.3)]
[Insomnia][Insomnia-1.0-REL.jar (1.0-REL)]
[LevelHead][LevelHead-8.2.2 (1.8.9).jar (8.2.2)]
[Danker's Skyblock Mod][MC.1.8.9.Danker.s.Skyblock.Mod.-.2.3.1.jar (2.3.1)]
[Mixmetica][Mixmetica-1.8.9-forge-2.1.0.jar (2.1.0)]
[NotEnoughUpdates][NotEnoughUpdates-2.3.3.jar (2.3.3+b23d0a27)]
[Optibye][Optibye-1.0.0-dep.jar (1.0.0)]
[OverflowAnimations][OverflowAnimations-1.8.9-forge-2.2.5.jar (2.2.5)]
[OverflowParticles][OverflowParticles-1.8.9-forge-1.0.2.jar (1.0.2)]
[PolyCrosshair][PolyCrosshair-1.8.9-forge-1.0.3.jar (1.0.3)]
[PolyNametag][PolyNametag-1.8.9-forge-1.0.7.jar (1.0.7)]
[Patcher][PolyPatcher-1.8.9-forge-1.10.1.jar (1.10.1)]
[PolySprint][PolySprint-1.8.9-forge-1.0.1.jar (1.0.1)]
[Popup Events][Popup Events-1.3.2 (1.8.9).jar (LOCAL)]
[Gravy's Reward Claim][RewardClaim-1.0.6.jar (1.0.6)]
[Scrollable Tooltips][Scrollable Tooltips-1.4 (1.8.9).jar (LOCAL)]
[SkyblockAddons Unofficial][SkyblockAddons-1.9.0-beta.16+366-for-MC-1.8.9.jar (1.9.0-beta.16+366)]
[SkyClient Updater][SkyClient-Updater-1.8.9-forge-1.3.10.jar (1.3.10)]
[SkyClientCosmetics][SkyClientCosmetics-1.8.9-forge-1.2.2.jar (1.2.2)]
[SkyGuide][SkyGuide (1.8.9-forge)-1.3.0+oneconfig.jar (1.3.0+oneconfig)]
[SkyHanni][SkyHanni-0.26.jar (0.26)]
[Skytils][Skytils-1.10.6.jar (1.10.6)]
[SoopyV2Forge][soopyv2forge-1.1.jar (1.1)]
[TNT Time][TNT Time-1.1 (1.8.9).jar (LOCAL)]
[VanillaHUD][VanillaHUD-1.8.9-forge-2.2.10.jar (2.2.10)]
```

### Operating System

Windows 10

**Comment by Fix3dll** [comment]
@Petar25 The specified features receive data via the action bar. In some cases (eg. "NOT ENOUGH MANA" text on actionbar) hypixel does not display mana by default. SBA includes a prediction code for such situations, but it does not provide 100% accurate values. There are many variables that we can or cannot access for prediction. It may be improved in the future, but it is not currently planned. 
tl;dr the specified features show the same values ​​as those written in the default actionbar.

If you think there is a different situation, you can reopen the issue with a more detailed explanation.
